### DISCUSSION TITLE
Migration from Edgerouter 4 to Cloud Gateway Ulta - IPTV

### DISCUSSION THREAD

**Discussion by wbvreeuwijk** [fabianishere/udm-iptv#312] [General]
Hi All,

I'm currently using an Edgerouter 4 and will move to the Cloud Gateway Ultra. In my current setup Internet and IPTV are split over two ethernet ports (eth1 - Internet and eth2 IPTV). 

For setting up IPTV on my Cloud Gateway Ultra I plan to use the udm-iptv script, but before I do I'd like to get an answer to the following questions:

- Based on what I have read so far it appears that both IPTV and Internet exits the gateway VLAN tagged instead of using two ethernet ports. Is this correct?
- I see VLAN661 used. Is this the VLAN ID of IPTV traffic exiting the Gateway?

Any assistance will be highly appreciated.

**Comment by dmouwes** [comment]
I'm using the cloud gateway ultra and the script doesn't work on it. I can currently only use the smart tv app to watch tv. Till there's a new script it will not work. Also IPTV Vlan is 2 not 661, that doesn't work either.

**Reply by Pingoe88** [reply]
if you have the kpn tv+ the box supports unicast. make a new vlan with igmp snooping on and the tv works. also go to settings > internet > Wan 1 > manual settings > under vlan id put 6 > under ipv4 connection select PPPoe > usernam/password is internet/internet > i also recomend to clone your mac of the experia box. or if your ultra already has a ip from the isp skip this. now put the port where the tv box is attached to to the IPTV vlan you created with igmp snooping. and you're done. it is a bit slower because its unicast. but at least you have everything set up till we get the script update

**Reply by dmouwes** [reply]
I dont have the tv+ box. Cloning the mac of experia box won't make a difference and never used one actually. I tried to connect vlan to the port, but doesn't make a difference. Only a new or adjusted script will do the trick.

**Reply by Pingoe88** [reply]
in that case you need the multicast and double vlan which sucks.
